iOS/Source/SplashViewController.swift

```diff
@@ -0,0 +1,157 @@
+import UIKit
+import CoreLocation
+import UserNotifications
+
+class SplashViewController: UIViewController {
```

Comment:
Since there isn't really a splash screen, I'd probably call this something more like `InitialViewController`